Assets/Scripts/EOSSessionsManager.cs

```diff
@@ -359,6 +365,70 @@ public class EOSSessionsManager : IEOSSubManager
+
+        /// <summary>
+        /// When sending or receiving Peer2Peer messages relating to Session status upkeep, always use this same channel.
+        /// This should be a distinct and unique channel, different than any other socket name utilized in the program, if possible.
+        /// </summary>
+        private const byte P2PSessionStatusUpdateChannel = 0xF;
+
+        /// <summary>
+        /// Messages about sessions should all start with this message, so that parsing the message for information is easy.
+        /// This is formatted in <see cref="P2PInformSessionMessageFormat"/> while sending messages.
+        /// </summary>
+        private const string P2PInformSessionMessageBase = "SESSIONINFORMATION";
+
+        /// <summary>
+        /// Messages about sessions follow this format, starting with <see cref="P2PInformSessionMessageBase"/>.
+        /// {0} - The back-end <see cref="Session.Id"/> of the session to message about.
+        /// {1} - The <see cref="ProductUserId"/> of the user sending the message.
+        /// {2} - Additional information about the message.
+        /// <see cref="P2PJoiningSessionMessageElement"/>
+        /// <see cref="P2PLeavingSessionMessageElement"/>
+        /// <see cref="P2PRefreshSessionMessageElement"/>
+        /// <see cref="P2PSessionOwnerDestroyedSessionMessageElement"/>
+        /// </summary>
+        private const string P2PInformSessionMessageFormat = P2PInformSessionMessageBase + " ({0}) ({1}) ({2})";
+
+        /// <summary>
+        /// Messages with this as the {2} parameter of <see cref="InformSessionOwnerMessageFormat"/> indicate a user is joining the session.
+        /// </summary>
+        private const string P2PJoiningSessionMessageElement = "JOIN";
+
+        /// <summary>
+        /// Messages with this as the {2} parameter of <see cref="P2PInformSessionMessageFormat"/> indicate a user is leaving the session.
+        /// </summary>
+        private const string P2PLeavingSessionMessageElement = "LEAVE";
+
+        /// <summary>
+        /// Messages with this as the {2} parameter of <see cref="P2PInformSessionMessageFormat"/> indicate that a user should re-acquire and refresh session information.
+        /// </summary>
+        private const string P2PRefreshSessionMessageElement = "REFRESH";
+
+        /// <summary>
+        /// Messages with this as the {2} parameter of <see cref="P2PInformSessionMessageFormat"/> indicate that the owner of a session has destroyed the session,
+        /// so members of the session should also remove themselves from the session.
+        /// </summary>
+        private const string P2PSessionOwnerDestroyedSessionMessageElement = "DESTROY";
+
+        /// <summary>
+        /// When subscribing to peer request connection messages, this Id is held as a way to later remove the subscription.
+        /// </summary>
+        private ulong P2PSessionPeerRequestConnectionNotificationId { get; set; }
```

Comment:
You asking this might suggest that we need to make the comment include this information.

This notification id is for all incoming requests. You don't need one of these per peer, just the one to allow peers to connect.

Reply:
That is a salient point - and clarifies my own understanding of the functionality.